src/Idris/Env.idr

```diff
@@ -0,0 +1,46 @@
+module Idris.Env
+
+--- All environment variables accessed by the compiler are enumerated in this module.
+
+import public Data.List.Elem
+
+import System
+
+||| Environment variable used by Idris2 compiler
+public export
+record EnvDesc where
+  constructor MkEnvDesc
+  name : String
+  help : String
+
+||| All environment variables used by Idris2 compiler
+public export
+envs : List EnvDesc
+envs = [
+         MkEnvDesc "EDITOR"        "Editor used in REPL :e command",
+         MkEnvDesc "IDRIS2_PREFIX" "Idris2 installation prefix",
+         MkEnvDesc "IDRIS2_PATH"   "Places Idris2 looks for import files",
+         MkEnvDesc "IDRIS2_DATA"   "Places Idris2 looks for data files",
+         MkEnvDesc "IDRIS2_LIBS"   "Places Idris2 looks for libraries (for code generation)",
+         MkEnvDesc "IDRIS2_CG"     "Idris2 CG directories",
```

Comment:
I believe `IDRIS2_CG` allows you to pick a backend

Reply:
Thanks!